integrations/venice-duckdb/src/main/java/com/linkedin/venice/sql/DuckDBDaVinciRecordTransformer.java

```diff
@@ -0,0 +1,130 @@
+package com.linkedin.venice.sql;
+
+import static com.linkedin.venice.sql.AvroToSQL.UnsupportedTypeHandling.FAIL;
+
+import com.linkedin.davinci.client.DaVinciRecordTransformer;
+import com.linkedin.davinci.client.DaVinciRecordTransformerResult;
+import com.linkedin.venice.utils.lazy.Lazy;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.function.BiConsumer;
+import org.apache.avro.Schema;
+import org.apache.avro.SchemaBuilder;
+import org.apache.avro.generic.GenericRecord;
+
+
+public class DuckDBDaVinciRecordTransformer extends DaVinciRecordTransformer<String, GenericRecord, GenericRecord> {
+  private static final String duckDBFilePath = "my_database.duckdb";
+  private static final String baseVersionTableName = "my_table_v";
+  private final String duckDBUrl;
+  private final String versionTableName;
+
+  // Can only support on primary key atm
+  private static final String primaryKey = "firstName";
+  private final Set<String> primaryKeys = new HashSet<>();
+
+  public DuckDBDaVinciRecordTransformer(int storeVersion, boolean storeRecordsInDaVinci, String baseDir) {
+    super(storeVersion, storeRecordsInDaVinci);
+    versionTableName = baseVersionTableName + storeVersion;
```

Comment:
Don't we get the store name passed down to us in the Transformer API? Hardcoding to "my_table" is fine in the prototype, but I think this will fail once we have multiple tables (unless we create separate DuckDB instances for each, but that is less ideal since it means the multiple tables could not be joined or otherwise queried together). Anyway, can be just a TODO for now.